sa/sa_test.go

```diff
@@ -992,6 +992,15 @@ func TestSetOrderProcessing(t *testing.T) {
 	test.AssertNotError(t, err, "GetOrder failed")
 	test.AssertEquals(t, *updatedOrder.Status, string(core.StatusProcessing))
 	test.AssertEquals(t, *updatedOrder.BeganProcessing, true)
+
+	// Try to set the same order to be processing again. We should get an error.
+	err = sa.SetOrderProcessing(context.Background(), order)
+	if err == nil {
+		t.Errorf("Set the same order processing twice. This should have been an error.")
+	}
```

Comment:
Small nit, I think this could be more succinct using the `test` package:
```
test.AssertError(t, err, "Set the same order processing twice. This should have been an error.")
```